daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-eventhubs.md

```diff
@@ -117,14 +117,37 @@ spec:
       value: "myeventhubstoragecontainer"
 ```
 
-## Sending multiple messages
+## Sending and receiving multiple messages
 
-Azure Event Hubs supports sending multiple messages in a single operation. To set the metadata for bulk operations, set the query parameters on the HTTP request or the gRPC metadata as documented [here]({{< ref pubsub_api >}})
+Azure Eventhubs supports sending and receiving multiple messages in a single operation using the bulk pub/sub API.
+
+### Configuring bulk publish
+
+To set the metadata for bulk publish operation, set the query parameters on the HTTP request or the gRPC metadata as documented [here]({{< ref pubsub_api >}}).
 
 | Metadata | Default |
 |----------|---------|
 | `metadata.maxBulkPubBytes` | `1000000` |
 
+### Configuring bulk subscribe
+
+When subscribing to a topic, you can configure `bulkSubscribe` options. Refer to [Subscribing messages in bulk]({{< ref "pubsub-bulk#subscribing-messages-in-bulk" >}}) for more details. Learn more about [the bulk subscribe API]({{< ref pubsub-bulk.md >}}).
+
+| Configuration | Default |
+|---------------|---------|
+| `maxMessagesCount` | `100` |
+| `maxAwaitDurationMs` | `10000` |
+
+## Configuring Checkpoint Frequency
```

Comment:
I do not understand what this section is trying to cover. Can you explain its usage some more?
It really need an example here on what you are trying to cover in this feature. Can you provide a more complex scenario and example?

Reply:
Let me add more context and an example for the same. Thanks for the suggestion

Reply:
Done. Please review